lib/proxy/provider.rb

```diff
@@ -24,7 +24,8 @@ def configure_plugin
     ::Proxy::BundlerHelper.require_groups(:default, bundler_group)
     after_activation
   rescue Exception => e
-    logger.error("Couldn't enable plugin #{plugin_name}: #{e}:#{e.backtrace.join("\n")}")
+    logger.error("Couldn't enable plugin #{plugin_name}: #{e}")
+    logger.debug("#{e}:#{e.backtrace.join("\n")}")
```

Comment:
In Foreman core, we decided to log backtraces at warn level rather than debug so they were still visible to most users without needing to switch logging settings.